desktop_app/src/backend/server/plugins/memory/index.ts

```diff
@@ -192,64 +192,6 @@ const memoryRoutes: FastifyPluginAsyncZod = async (fastify) => {
       return { success: true, count };
     }
   );
-
-  // Legacy endpoints for backward compatibility
-  fastify.get(
-    '/api/memory',
-    {
-      schema: {
-        operationId: 'getMemory',
-        description: 'Get the current user memory (legacy format)',
-        tags: ['Memory'],
-        response: {
-          200: LegacyMemoryResponseSchema,
-        },
-      },
-    },
-    async (_request, _reply) => {
-      const content = await MemoryModel.getMemories();
-      return { content };
-    }
-  );
-
-  fastify.put(
-    '/api/memory',
-    {
-      schema: {
-        operationId: 'updateMemory',
-        description: 'Update the current user memory (legacy format)',
-        tags: ['Memory'],
-        body: WriteMemorySchema,
-        response: {
-          200: z.object({ success: z.boolean() }),
-          400: z.object({
-            error: z.string(),
-            message: z.string(),
-          }),
-        },
-      },
-    },
-    async ({ body }, reply) => {
-      // Validate that content is not empty
-      if (!body.content || body.content.trim() === '') {
-        return reply.code(400).send({
-          error: 'Invalid memory content',
-          message: 'Memory content cannot be empty',
-        });
-      }
-
-      await MemoryModel.writeMemories(body.content.trim());
-
-      // Emit WebSocket event for memory update
-      const memories = await MemoryModel.getAllMemories();
-      websocketService.broadcast({
-        type: 'memory-updated',
-        payload: { memories },
-      });
-
-      return { success: true };
-    }
-  );
```

Comment:
the codegen'd sdk methods for these two endpoints (which are commented as "legacy") are no longer used